src/mode/modeNormal.ts

```diff
@@ -23,8 +23,10 @@ export class NormalMode extends Mode {
         "gg" : async (c) => { return c.firstLineNonBlankChar().move(); },
         "G" : async (c) => { return c.lastLineNonBlankChar().move(); },
         "w" : async (c) => { return c.wordRight().move(); },
+        "W" : async (c) => { return c.WORDRight().move(); },

```

Comment:
Yeah, I'm not usually a fan either. Though I figured in this case it would be the most obvious vocabulary for someone familiar with vim.

I took a look at various other projects to see what they use, and I think I like "Big Word" the best. Still short, and the connection can be made to WORD pretty easily and recalled easily. I'll make an update to the pull request.